File: WES/ClientApp/src/app/principal/usuarios/models/usuario-model.ts
export interface Usuario {
  Id: number
  Email: string;
  Password: string;
  Nombre: string;
  Apellidos: string;
  Telefono: number;
  FechaNacimiento: Date;
  Sexo: TipoSexo;
  SexoTexto: number;
  Rol: number;
  RolTexto: string;
}

enum TipoSexo {
  Mujer = 0,
  Hombre = 1,
}

File: WES/Models/Usuario/UsuarioModel.cs
namespace WES.Models
{
    public class UsuarioModel
    {
        public int Id { get; set; }
        public string? Email { get; set; }
        public string? Password { get; set; }
        public string? Nombre { get; set; }
        public string? Apellidos { get; set; }
        public int Telefono { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public int Sexo { get; set; }
        public string? SexoTexto { get; set; }
        public int Rol { get; set; }
        public string? RolTexto { get; set; }
    }
}
